velox/dwio/parquet/tests/reader/ParquetTableScanTest.cpp

```diff
@@ -836,6 +846,26 @@ TEST_F(ParquetTableScanTest, timestampPrecisionMicrosecond) {
   assertEqualResults({expected}, result.second);
 }
 
+TEST_F(ParquetTableScanTest, testColumnNotExists) {
```

Comment:
Will you be able to add tests for ARRAY and MAP types too?

Reply:
Already added.